<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_random_or_input">
       <property name="text">
        <string>Random fill or fill yourself?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkbox_random">
       <property name="text">
        <string>Random</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkbox_input">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_enter_seed">
       <property name="text">
        <string>Enter random seed or input:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineedit_seed_or_input"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushbutton_place_number">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Place</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdnumber_minutes">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>380</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdnumber_seconds">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton_pause">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton_help">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>help</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textbrowser_rules">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton_start">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushbutton_reset">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_general">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>482</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
